docs/API/Observability/TelemetryEventListener.md

```
@@ -0,0 +1,97 @@
+
+## Namespace
+
+`OpenFGA\Observability`
+
+## Source
+
+[View source code](https://github.com/evansims/openfga-php/blob/main/src/Observability/TelemetryEventListener.php)
+
+## Methods
+
+#### onHttpRequestSent
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)